src/providers/wfs/qgswfsdataitems.cpp

```diff
@@ -192,6 +195,84 @@ void QgsWfsRootItem::newConnection()
 }
 #endif
 
+
+//////
```

Comment:
?

Reply:
Fixed - it was just used throughout this file as an indicator of "new class"